Description: use local intersphinx inventory
 Use the offline copy under debian/ instead downloading it.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Forwarded: not-needed
Last-Update: 2015-01-05

---

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -51,7 +51,7 @@ extensions = [
 ]
 
 intersphinx_mapping = {
-    'python': ('https://docs.python.org/', None),
+    'python': ('https://docs.python.org/', '../debian/python.inv'),
     'greenlet': ('https://greenlet.readthedocs.io/en/latest/', None),
     'zopeevent': ('https://zopeevent.readthedocs.io/en/latest/', None),
     'zopecomponent': ('https://zopecomponent.readthedocs.io/en/latest/', None),
